THIRD-PARTY.md

```diff
@@ -7,46 +7,45 @@ into parts of OSL.
 
 In no particular order:
 
--------------------------------------------------------------------------
+---
 
 BSD 3-Clause License
 SPDX-License-Identifier: BSD-3-Clause
 
-* OpenImageIO Copyright (c) Larry Gritz, et al.
+- OpenImageIO Copyright (c) Larry Gritz, et al.
   https://github.com/OpenImageIO/oiio
 
   Many bits of code flow back and forth between OIIO and OSL, especially
   parts of the CMake build system.
 
-* OpenEXR, Copyright (c) Contributors to the OpenEXR Project.
+- OpenEXR, Copyright (c) Contributors to the OpenEXR Project.
   https://github.com/AcademySoftwareFoundation/OpenEXR
 
   OSL replicates and modifies several files from the "Imath" portion of
   OpenEXR. These are all found in src/include/OSL/Imathx.
 
-* Qt, Copyright (c) The Qt Company Ltd.
+- Qt, Copyright (c) The Qt Company Ltd.
   http://doc.qt.io/
 
```

Comment:
Please revert all *changes* to THIRD-PARTY.md.

Don't change anything that's already there. Only add what you need for your particular change. If you think something else in this file is incorrect, let's handle that in a separate PR.

Reply:
Sorry about that @lgritz 
these lines are automatically corrected by VSCode autoformat of `md` files
was not intentionally

Reply:
thanks